app/views/admin/neighbourhoods/show.html.erb

```diff
@@ -31,7 +31,13 @@
 <% if @neighbourhood.address_partners.present? %>
 <ul class="list-group">
   <% @neighbourhood.address_partners.order(:name).each do |partner| %>
-  <li class="list-group-item"><%= link_to partner.name, edit_admin_partner_path(partner) %></li>
+  <li class="list-group-item">
+    <% if current_user.partner_admin? -%>
```

Comment:
Unfortunately I think this will only check if you're a partner admin of any partner, not this specific one. I think based on the user stories we generated yesterday with kim the logic here needs to be that you should only be able to edit this partner if you own it, it is in your neighbourhood, or if you are a root. This should probably be something we mandate at a user model level, through something like can_edit_partner(partner) or similar?

Reply:
I have updated this to use the same policy check that "update partner" action uses.